Jenkinsfile.central

```diff
@@ -0,0 +1,73 @@
+#!groovy
+
+node{
+   properties(
+       [parameters(
+           [string(defaultValue: 'develop', description: 'The SHA for the commit you wish to deploy.', name: 'GIT_COMMIT', trim: false),
+            string(defaultValue: '', description: 'The tag of the current release.', name: 'RELEASE_TAG', trim: false)]
+        )]
+    )
+    
+    try{
+        env.AWS_DEFAULT_REGION = "us-east-1"
+
+        // Checkout the proper revision into the workspace.
+        stage('checkout') {
+          checkout([
+            $class: 'GitSCM',
+            branches: [[name: params.GIT_COMMIT]],
+            extensions: scm.extensions + [[$class: 'CloneOption', noTags: false, reference: '', shallow: false]],
+            userRemoteConfigs: scm.userRemoteConfigs
+          ])
+        }
+
+        env.RF_SETTINGS_BUCKET = 'rasterfoundry-testing-config-us-east-1'
+
+        stage('bootstrap') {
+          sh """#!/bin/bash
+            aws s3 cp "s3://${RF_SETTINGS_BUCKET}/.env" ".env"
```

Comment:
Within the existing framework of our scripts, I can't think of a place to put this.